flang/test/Lower/OpenMP/Todo/masked-directive.f90

```diff
@@ -0,0 +1,65 @@
+
+! This test checks lowering of OpenMP masked Directive.
+
+// RUN: not flang-new -fc1 -emit-fir -fopenmp %s 2>&1 | FileCheck %s
+
+subroutine test_masked()
+  integer :: c = 1
+  // CHECK: not yet implemented: Unhandled Directive masked
```

Comment:
Nit: Use "!" rather than "//" for Fortran comments.
```suggestion
  ! CHECK: not yet implemented: Unhandled Directive masked
```

Reply:
Please fix these comments before merging.

Reply:
Yeah, thanks. Corrected. Its a copy paste mistake. I noticed a few more examples in the same folder with the same mistake. Will correct those in a separate PR.